codebase-pipelines/tests/unit.tftest.hcl

```diff
@@ -84,10 +91,9 @@ variables {
       }
     }
   }
-  application               = "my-app"
-  codebase                  = "my-codebase"
-  repository                = "my-repository"
-  additional_ecr_repository = "my-additional-repository"
+  application = "my-app"
+  codebase    = "my-codebase"
+  repository  = "my-repository"
```

Comment:
It is very sad that we cannot disable this aligning of equals signs, it makes so much noise in our pull requests.

It looks like [the last attempt to request this](https://github.com/hashicorp/terraform/issues/33281) was shut down almost immediately. I might re-raise it when I have the energy.